Tests/IteratorsTests/PeekingIteratorTests.swift

```diff
@@ -13,4 +13,13 @@ final class PeekingIteratorTests: XCTestCase {
         XCTAssertEqual(nil, iterator.peek())
         XCTAssertEqual(nil, iterator.next())
     }
+
+    func testPeekingIterator_WithoutPeek() {
+        // Make sure that iterator does not require peek to be called first.
```

Comment:
I think the above test name suggestion takes care of this comment.

```suggestion
```

Reply:
Done.